Source/Demos/Demo.Particles/Game1.cs

```diff
@@ -28,6 +28,7 @@ public Game1()
             Content.RootDirectory = "Content";
             IsMouseVisible = true;
             Window.AllowUserResizing = true;
+            Window.Position = Point.Zero;
```

Comment:
Okay good to know. Having the window in the top left corner feels a little odd but obviously it's not really doing any harm. It is what it is I guess.